frontends/precinct-scanner/src/screens/scan_warning_screen.tsx

```diff
@@ -118,26 +125,27 @@ function UndervoteWarningScreen({
         <h1>Review Your Ballot</h1>
         {blankContestNames.length > 0 && (
           <Text small>
-            You did not vote for {toSentence(blankContestNames, ', ', ' or ')}.
+            No votes detected for: {toSentence(blankContestNames, ', ', ' or ')}
+            .
           </Text>
         )}
         {partiallyVotedContestNames.length > 0 && (
           <Text small>
-            You can vote for more people for{' '}
+            You may vote for more candidates for:{' '}
```

Comment:
It's not always going to be a candidate as it could be a ballot measure. We should use more generic language.

Reply:
Are there ballot measures that allow you to vote for more than one choice? This case only shows for vote-for-n contests where n>1

Reply:
Thanks for clarifying that. We should be ok in that case.

Reply:
@mattroe - I had the same thought at first, then realized that only candidate contests can have more than one vote. yesno contests and mississippi either neither don't apply.